trinity/utils/chains.py

```diff
@@ -107,19 +107,23 @@ def load_nodekey(nodekey_path: str) -> PrivateKey:
 
 
 class ChainConfig:
-    _data_dir = None
-    _nodekey_path = None
+    _data_dir: str = None
+    _nodekey_path: str = None
```

Comment:
Maybe some of these could be [PurePath](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath) objects?

Reply:
https://github.com/ethereum/py-evm/issues/725